stl_to_obj/stl_to_tscn.py

```diff
@@ -0,0 +1,209 @@
+import aspose.threed as a3d
+import argparse
+import os
+import bpy
+import json
+import shutil
+import textures as Textures # custom module created to store texture info
```

Comment:
```suggestion
import argparse
import os
import json
import shutil

import aspose.threed as a3d
import bpy

import textures as Textures # custom module created to store texture info
```
You might consider grouping imports as suggested to follow [the corresponding PEP8 guidelines](https://peps.python.org/pep-0008/#imports).

For imports, guidelines are to group imports as follows, with whitespace between import groups:

1. Standard library imports (Python libraries that are built-in)
2. Related third party imports (dependencies)
3. Local application/library specific imports (custom modules housed locally)